source/install/config-ssl-http2-apache2.rst

```diff
@@ -0,0 +1,64 @@
+.. _config-ssl-http2-apache2:
+
```

Comment:
Can you add this note:

```
.. important:: This unofficial guide is maintained by the Mattermost community and this deployment configuration is not yet officially supported by Mattermost, Inc. Community testing, feedback and improvements are welcome and greatly appreciated. You can `edit this page on GitHub <https://github.com/mattermost/docs/blob/master/source/install/config-ssl-http2-apache2.rst>`_.
```
This note is slightly different than the one for `config-proxy-apache2.rst` so make sure you paste this one in.